### DISCUSSION TITLE
When poll closes, recommend Meeting/Event day & time and allow click OR auto-send Calendar invite

### DISCUSSION THREAD

**Discussion by jcolinger3** [lukevella/rallly#692] [Ideas]
When Poll closes, recommend Meeting/Event day & time and allow click OR auto-send Calendar invite

When creating a Poll, allow the creator to select "Auto-invite attendees to Calendar event/meeting upon Poll close based on best day/time" OR "Recommend best day(s)/times(s) with easy click to create Calendar event/meeting"

I feel that with the addition of this feature (or something like it), it would take Rallly to a very elegant and useful level.  The Polls are certainly helpful, but still requires some manual steps to really get to the actual meeting.

**Comment by Kruptein** [comment]
This is similar to https://github.com/lukevella/rallly/discussions/627, which seemed to be close to done in March, but is a bit stuck at the moment.
